lib/ide/ast/macros/src/token.rs

```diff
@@ -0,0 +1,98 @@
+use crate::prelude::*;
+
+use macro_utils::path_segment_generic_args;
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::Expr;
+use syn::GenericArgument;
+use syn::PathSegment;
+use syn::Token;
+use syn::punctuated::Punctuated;
+
+/// Generates `Tokenizer` that just panics when called.
+pub fn no_tokenizer
```

Comment:
`panicking_tokenizer`?
I don't like "no_" as that suggests to me as compile-time error is user tries to call it.